packages/slate-editor/src/lib/scrollIntoView.ts

```diff
@@ -44,11 +44,6 @@ export function scrollIntoView(
         return;
     }
 
-    if (isChildAboveVisibleArea) {
-        const y = parent.scrollTop + elementTop - minTop;
-        scrollTo(parent, parent.scrollLeft, y);
-    } else if (isChildBelowVisibleArea) {
-        const y = parent.scrollTop + elementTop + elementHeight - parentHeight + minBottom;
-        scrollTo(parent, parent.scrollLeft, y);
-    }
```

Comment:
I believe that the change to this part of logic has introduced a major UX regression for interactions with the content. 

It now always jumps the viewport to put the cursor to the top of the viewport. That makes the editing experience suboptimal (to say the least). I didn't see any other editor out there behaving this way.


https://user-images.githubusercontent.com/370680/165104597-96a64eb5-4fd1-466c-b9d1-9ca3a76c74c9.mp4